### ISSUE TITLE
Sigmoid result all zeros in the exported onnx model

### ISSUE THREAD

**Issue by ZhangTianrong** [pytorch/pytorch#70515]
### 🐛 Describe the bug

I am trying to export a model to onnx but the output is wrong. The output has correct size but is all zeros which is nearly impossible for a sigmoid function.

![image](https://user-images.githubusercontent.com/20651728/147723398-93f1ee88-6cd5-4d2d-80de-14b34ef1b12c.png)

Interestingly, when I looked into the value of node 705, it's correct. The involved pytorch module here has a forward function like this:

```python
def forward(self, x):
    x = self.conv1(x)
    x = self.conv_bn1(x)
    x = self.relu1(x)
    x = self.conv2(x)
    x = self.conv_bn2(x)
    x = self.relu2(x)
    x = self.conv3(x)
    x = torch.sigmoid(x)
    return x
```

I tried removing the sigmoid and then I can get the correct result. That means there might be something wrong with the sigmoid itself, but when I tested with a trivial model with only a sigmoid operation, there is no such problem.
I'm not sure it this helps, but I export the onnx model in the following way:

```python
torch.onnx.export(
                    self.net,
                    inp,
                    "det.onnx",
                    export_params=True,
                    opset_version=11,
                    verbose=True,
                    do_constant_folding=True, 
                    input_names = ['input'],
                    output_names = ['output'],
                    dynamic_axes={
                        'input' : {
                            0 : 'batch_size',
                            2 : 'height_in',
                            3 : 'width_in'
                        }
                    }
                )
```

The exportation log is pretty long, but I guess only the last lines matter.

```
  %705 : Float(*, 1, *, *, strides=[458752, 458752, 896, 1], requires_grad=0, device=cuda:0) = onnx::ConvTranspose[dilations=[1, 1], group=1, kernel_shape=[2, 2], pads=[0, 0, 0, 0], strides=[2, 2]](%704, %head.binarize.conv3.weight, %head.binarize.conv3.bias)
  %output : Float(*, 1, *, *, strides=[458752, 458752, 896, 1], requires_grad=0, device=cuda:0) = onnx::Sigmoid(%705) 
  return (%output)
```

Maybe this is an onnx issue, given the that graph is correct? If that is the case then I will open an issue there.

### Versions

Collecting environment information...
PyTorch version: 1.10.1+cu113
Is debug build: False
CUDA used to build PyTorch: 11.3
ROCM used to build PyTorch: N/A

OS: Ubuntu 20.04.3 LTS (x86_64)
GCC version: (Ubuntu 9.3.0-17ubuntu1~20.04) 9.3.0
Clang version: Could not collect
CMake version: Could not collect
Libc version: glibc-2.31

Python version: 3.9.7 (default, Sep 16 2021, 13:09:58)  [GCC 7.5.0] (64-bit runtime)
Python platform: Linux-5.10.60.1-microsoft-standard-WSL2-x86_64-with-glibc2.31
Is CUDA available: True
CUDA runtime version: Could not collect
GPU models and configuration: GPU 0: NVIDIA GeForce RTX 3050 Ti Laptop GPU
Nvidia driver version: 510.06
cuDNN version: Could not collect
HIP runtime version: N/A
MIOpen runtime version: N/A

Versions of relevant libraries:
[pip3] numpy==1.21.4
[pip3] torch==1.10.1+cu113
[pip3] torchaudio==0.10.1+cu113
[pip3] torchvision==0.11.2+cu113
[conda] blas                      1.0                         mkl  
[conda] mkl                       2021.4.0           h06a4308_640  
[conda] mkl-service               2.4.0            py39h7f8727e_0  
[conda] mkl_fft                   1.3.1            py39hd3c417c_0  
[conda] mkl_random                1.2.2            py39h51133e4_0  
[conda] numpy                     1.21.4                   pypi_0    pypi
[conda] numpy-base                1.21.2           py39h79a1101_0  
[conda] torch                     1.10.1+cu113             pypi_0    pypi
[conda] torchaudio                0.10.1+cu113             pypi_0    pypi
[conda] torchvision               0.11.2+cu113             pypi_0    pypi

**Comment by ZhangTianrong** [comment]
Seems that onnxruntime is not able to perform the sigmoid in a numerically stable way in my case.
